### PR TITLE
WIP: Pinpoint guest blog

### PR DESCRIPTION
**Pull request by spara**

### CODE REVIEW

File: content/blog/pinpoint-kubernetes/index.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
+---
+title: "How Pinpoint Manages Kubernetes with Pulumi"
+date: 2020-08-14
+meta_desc: "Pinpoint uses Pulumi to streamline deployments and scale Kubernetes"
+meta_image: meta.png
+authors:
+    - sophia-parafina
```
Comment (isaac-pulumi) [inline content/blog/pinpoint-kubernetes/index.md:7]: Can we add the Pinpoint team names here?

File: content/blog/pinpoint-kubernetes/index.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
+---
```
Comment (isaac-pulumi) [inline content/blog/pinpoint-kubernetes/index.md:1]: We need to change the URL to www.pulumi.com/blog/pinpoint (that is what is going in the Press Release that we sent to CNCF.

File: content/blog/pinpoint-kubernetes/index.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+date: 2020-08-14
+meta_desc: "Pinpoint uses Pulumi to streamline deployments and scale Kubernetes"
+meta_image: meta.png
+authors:
+    - sophia-parafina
+tags:
+    - Pinpoint
+    - Kubernetes
+---
+*This guest blog was contributed by Andrew Kunzel and Michael Goode. Andresis the Director of Backend Engineering and Michael is a Platform Operations Engineer at Pinpoint.*
```
Comment (isaac-pulumi) [inline content/blog/pinpoint-kubernetes/index.md:12]: > 
```suggestion
*This guest blog was contributed by Andrew Kunzel and Michael Goode. Andrew is the Director of Backend Engineering and Michael is a Platform Operations Engineer at Pinpoint.*
```

File: content/blog/pinpoint-kubernetes/index.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+
+<!--more-->
+
+Pulumi affords us the ability to streamline the development of our cloud infrastructure with open source libraries that integrate deeply into AWS service APIs and support many popular languages. This introduces powerful, high-level constructs like for-loops and strong type-checking alongside the more rigid declarative approach mandated by other available tools. And it’s not just limited to hardware! We also use Pulumi to substantially reduce the overhead required to deploy critical services that run on Kubernetes itself, which helps to consolidate our deployment configuration even further and makes updating these services a breeze.
+
+In this blog post, we’ll share some simple and intuitive ways in which Pulumi has simplified our approach to infrastructure-as-code.
+
+## Spot Instances
```
Comment (isaac-pulumi) [inline content/blog/pinpoint-kubernetes/index.md:22]:
```suggestion
## Managing Costs with Spot Instances
```

File: content/blog/pinpoint-kubernetes/index.md [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+meta_image: meta.png
+authors:
+    - sophia-parafina
+tags:
+    - Pinpoint
+    - Kubernetes
+---
+*This guest blog was contributed by Andrew Kunzel and Michael Goode. Andresis the Director of Backend Engineering and Michael is a Platform Operations Engineer at Pinpoint.*
+
+At Pinpoint, Kubernetes is the most powerful tool in our arsenal. It allows us to deploy and rapidly scale our applications with speed and efficiency that continues to delight our customers. In recent years, managed services like AWS EKS have made it easier than ever to leverage the power of Kubernetes in even the smallest of organizations. Yet even with these new conveniences, managing all of this infrastructure can be a daunting task. Right out of the gate, we knew that we wanted to avoid the burden of maintaining repositories full of home-brewed deployment scripts and domain-specific languages like YAML.
```
Comment (isaac-pulumi) [inline content/blog/pinpoint-kubernetes/index.md:14]: [https://pinpoint.com/](url) -- backlink the text 'Pinpoint'

File: content/blog/pinpoint-kubernetes/index.md (hunk 1 above)
Comment (spara) [reply content/blog/pinpoint-kubernetes/index.md:7]: yes, I'll add them

**Pull request closed without merging**